bash/relver

```diff
@@ -9,6 +9,13 @@ cd ..
 
 VER="Uncontrolled";
 
+get_tags() {
+    if ! git config remote.upstream.url > /dev/null; then
```

Comment:
Not sure about this. Does the condition need to be enclosed by brackets? That is,
    if [ ! git config remote.upstream.url > /dev/null; ] then

Also, add 2>&1 at the end?